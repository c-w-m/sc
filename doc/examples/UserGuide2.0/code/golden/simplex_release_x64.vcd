$date
     Oct 23, 2015       17:18:57
$end

$version
 SystemC 2.3.1-Accellera --- Oct 22 2015 09:01:36
$end

$timescale
     1 ps
$end

$scope module SystemC $end
$var wire    1  aaaaa  clock       $end
$var wire    1  aaaab  timeout       $end
$var wire    1  aaaac  start       $end
$var wire   32  aaaad  packet1.info [31:0]  $end
$var wire   32  aaaae  packet1.seq [31:0]  $end
$var wire   32  aaaaf  packet1.retry [31:0]  $end
$var wire   32  aaaag  packet2.info [31:0]  $end
$var wire   32  aaaah  packet2.seq [31:0]  $end
$var wire   32  aaaai  packet2.retry [31:0]  $end
$var wire   32  aaaaj  packet3.info [31:0]  $end
$var wire   32  aaaak  packet3.seq [31:0]  $end
$var wire   32  aaaal  packet3.retry [31:0]  $end
$var wire   32  aaaam  packet4.info [31:0]  $end
$var wire   32  aaaan  packet4.seq [31:0]  $end
$var wire   32  aaaao  packet4.retry [31:0]  $end
$var wire   32  aaaap  dout [31:0]  $end
$upscope $end
$enddefinitions  $end

$comment
All initial values are dumped below at time 0 sec = 0 timescale units.
$end

$dumpvars
1aaaaa
0aaaab
0aaaac
b100110 aaaad
b1 aaaae
b0 aaaaf
b0 aaaag
b0 aaaah
b0 aaaai
b0 aaaaj
b0 aaaak
b0 aaaal
b100110 aaaam
b0 aaaan
b0 aaaao
b0 aaaap
$end

#10000
0aaaaa

#20000
1aaaaa

#30000
0aaaaa

#40000
1aaaaa

#50000
0aaaaa

#60000
1aaaaa

#70000
0aaaaa

#80000
1aaaaa

#90000
0aaaaa

#100000
1aaaaa
b1 aaaaf
b1 aaaan
b1 aaaao

#105000
b1 aaaah
b10 aaaai
b1 aaaak
b10 aaaal
b100110 aaaap

#110000
0aaaaa

#120000
1aaaaa
b10000010101101 aaaad
b10 aaaae
b0 aaaaf
b10000010101101 aaaam
b10 aaaan
b0 aaaao

#130000
0aaaaa

#135000
b10 aaaah
b11 aaaai
b10 aaaak
b11 aaaal
b10000010101101 aaaap

#140000
1aaaaa
b111011110010100 aaaad
b11 aaaae
b111011110010100 aaaam
b11 aaaan

#150000
0aaaaa
b11 aaaah
b100 aaaai
b11 aaaak
b100 aaaal
b111011110010100 aaaap

#160000
1aaaaa
b10001110110 aaaad
b100 aaaae
b10001110110 aaaam
b100 aaaan

#165000
b100 aaaah
b101 aaaai
b100 aaaak
b101 aaaal
b10001110110 aaaap

#170000
0aaaaa

#180000
1aaaaa
b11111000110001 aaaad
b101 aaaae
b11111000110001 aaaam
b101 aaaan

#190000
0aaaaa

#195000
b101 aaaah
b110 aaaai
b101 aaaak
b110 aaaal
b11111000110001 aaaap

