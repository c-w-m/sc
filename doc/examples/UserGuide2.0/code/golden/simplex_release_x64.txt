===============================================================================
File: simplex_release_x64.txt
set SC_SIGNAL_WRITE_CHECK=DISABLE
std::srand(0)
===============================================================================

        SystemC 2.3.1-Accellera --- Oct 22 2015 09:01:36
        Copyright (c) 1996-2014 by all Contributors,
        ALL RIGHTS RESERVED

Info: (I804) /IEEE_Std_1666/deprecated: sc_sensitive_pos is deprecated use sc_sensitive << with pos() instead
Generate:Sending Data Value = 38

Info: (I804) /IEEE_Std_1666/deprecated: positional binding using << or , is deprecated, use () instead.
Channel:Received packet seq no. = 0
Channel: Random number = 7719
Channel:Received Ack for packet = 0
Display:Data Value Received, Data = 0
Transmit:Sending packet no. 1
Channel:Received packet seq no. = 1
Channel: Random number = 2437
Timer: timer start detected

Info: (I702) default timescale unit used for tracing: 1 ps (simplex.vcd)
Receiver: got packet no. = 0
Transmit:Sending packet no. 1
Channel:Received packet seq no. = 1
Channel: Random number = 8855
Timer: timer start detected
Receiver: got packet no. = 1
Display:Data Value Received, Data = 38
Channel:Received Ack for packet = 1
Generate:Sending Data Value = 8365
Transmit:Sending packet no. 2
Channel:Received packet seq no. = 2
Channel: Random number = 32285
Timer: timer start detected
Receiver: got packet no. = 2
Display:Data Value Received, Data = 8365
Channel:Received Ack for packet = 2
Generate:Sending Data Value = 30612
Transmit:Sending packet no. 3
Channel:Received packet seq no. = 3
Channel: Random number = 5853
Timer: timer start detected
Receiver: got packet no. = 3
Display:Data Value Received, Data = 30612
Channel:Received Ack for packet = 3
Generate:Sending Data Value = 1142
Transmit:Sending packet no. 4
Channel:Received packet seq no. = 4
Channel: Random number = 281
Timer: timer start detected
Receiver: got packet no. = 4
Display:Data Value Received, Data = 1142
Channel:Received Ack for packet = 4
Generate:Sending Data Value = 15921
Transmit:Sending packet no. 5
Channel:Received packet seq no. = 5
Channel: Random number = 8945
Timer: timer start detected
Receiver: got packet no. = 5
Display:Data Value Received, Data = 15921
Channel:Received Ack for packet = 5

Info: (I804) /IEEE_Std_1666/deprecated: You can turn off warnings about
             IEEE 1666 deprecated features by placing this method call
             as the first statement in your sc_main() function:

  sc_core::sc_report_handler::set_actions( "/IEEE_Std_1666/deprecated",
                                           sc_core::SC_DO_NOTHING );

Press any key to continue . . .